workflows/src/main/kotlin/net/cordapp/demo/obligation/messages/DeleteObligationRequestMessage.kt

```diff
@@ -0,0 +1,5 @@
+package net.cordapp.demo.obligation.messages
+
+import java.util.UUID
+
+data class DeleteObligationRequestMessage(val id: UUID)
```

Comment:
I think you can create a single file ObligationMessages.kt and create all data classes there